### ISSUE TITLE
Maximum baud rate

### ISSUE THREAD

**Issue by anonymix007** [Noltari/pico-uart-bridge#23]
What will be the maximum achievable baud rate?  I've seen reports of ~1 MBaud, but for my application 3 or more will be needed. Can it be achieved on RP2040?

**Comment by xxxajk** [comment]
Providing that you have proper wiring, yeah... But you really want to wait until bug #20 is fixed with https://github.com/Noltari/pico-uart-bridge/pull/24 or just use my branch until that point. *technically* it should max out at  961.6 kBaud, however there may be ways to overclock bits and pieces ;-)

**Comment by csarn** [comment]
I tried it with a short loopback wiring (connecting both pico uarts together), and had increasingly severe transmission errors starting at 1MBaud. Then I pulled the changes from the mentioned #24 (or rather #25), and had zero transmission errors (I tried up to 5Mbaud).

**Comment by xxxajk** [comment]
Yeah, remembering to have the pullup enabled is the trick to that working in my original patch.
